yaml/pois.yaml

```diff
@@ -260,6 +260,9 @@ filters:
   - filter: {tourism: winery}
     min_zoom: GREATEST(14, LEAST(zoom + 2.85, 17))
     output: {kind: winery}
+  - filter: {tourism: gallery}
+    min_zoom: GREATEST(15, LEAST(zoom, 17))
```

Comment:
I suspect this is more like museums, merged with your logic something like?

`min_zoom: GREATEST(15, LEAST(zoom + 1.43, 17))`